applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -471,8 +477,32 @@ pub async fn command_runner(
                 let utxos = output_service.get_unspent_outputs().await?;
                 let count = utxos.len();
                 let sum: MicroTari = utxos.iter().map(|utxo| utxo.value).sum();
-                for (i, utxo) in utxos.iter().enumerate() {
-                    println!("{}. Value: {} {}", i + 1, utxo.value, utxo.features);
+                if parsed.args.is_empty() {
+                    for (i, utxo) in utxos.iter().enumerate() {
+                        println!("{}. Value: {} {}", i + 1, utxo.value, utxo.features);
+                    }
+                } else if let ParsedArgument::CSVFile(file) = parsed.args[0].clone() {
+                    let factory = PedersenCommitmentFactory::default();
+                    let file = File::create(file).map_err(|e| CommandError::File(e.to_string()))?;
+                    let mut csv_file = LineWriter::new(file);
+                    csv_file
+                        .write_all(b"\"#\",\"Value (uT)\",\"Spending Key\",\"Commitment\",\"Features\"\n")
```

Comment:
Might be a little neater to use `writeln!` and `r##"(your string here)"##` instead of escaping the quotes
```suggestion
                        writeln!(csv_file, r##""#","Value (uT)","Spending Key","Commitment","Features""##)
```